wolfcrypt/test/test.c

```diff
@@ -36,7 +36,8 @@
     #define HAVE_WOLFCRYPT_TEST_OPTIONS
 #endif
 
-#ifdef HAVE_WOLFCRYPT_TEST_OPTIONS
+#if defined(HAVE_WOLFCRYPT_TEST_OPTIONS) || \
```

Comment:
Getting these build errors:

```
wolfcrypt/test/test.c:20566:12: error: unused function 'ecc_test_vector' [-Werror,-Wunused-function]
static int ecc_test_vector(int keySize)
           ^
wolfcrypt/test/test.c:21537:12: error: unused function 'ecc_test_key_decode' [-Werror,-Wunused-function]
static int ecc_test_key_decode(WC_RNG* rng, int keySize)
           ^
wolfcrypt/test/test.c:21618:12: error: unused function 'ecc_test_key_gen' [-Werror,-Wunused-function]
static int ecc_test_key_gen(WC_RNG* rng, int keySize)
           ^
wolfcrypt/test/test.c:22216:12: error: unused function 'ecc_point_test' [-Werror,-Wunused-function]
static int ecc_point_test(void)
           ^
wolfcrypt/test/test.c:22459:12: error: unused function 'ecc_sig_test' [-Werror,-Wunused-function]
static int ecc_sig_test(WC_RNG* rng, ecc_key* key)
           ^
wolfcrypt/test/test.c:22512:12: error: unused function 'ecc_exp_imp_test' [-Werror,-Wunused-function]
static int ecc_exp_imp_test(ecc_key* key)
           ^
wolfcrypt/test/test.c:22642:12: error: unused function 'ecc_mulmod_test' [-Werror,-Wunused-function]
static int ecc_mulmod_test(ecc_key* key1)
           ^
```

Reply:
tied to use clang to verify unused function error. most of them fixed.